docs/getting-started/cli/subsembly-spec.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Spec files
+---
+
+# subsembly spec
+
+### Overview
+
+A command used for generating chain specification files and converting them into raw.
+
+### Syntax
+
+```text
+subsembly spec [to] [src] [raw] [wasm]
+```
+
+#### Parameters
+
+* **`--to`** - \[optional\] initialisation path for the new chain specification file
+* **`--src`** - \[optional\] source specification file to convert to raw, defaults to current directory
+* **`--raw`** - \[optional\] path to save converted raw chain specification file. Defaults to **`{srcPath}/raw-chain-spec.json`**
+* **`--wasm`** - \[optional\] path to the compiled hex encoded **`wasm`** binary. Defaults to **`build/subsembly-wasm`**
+
+### Examples
```

Comment:
I think that we should add examples of the output and separate each command in a separate inlined box